static/collection.js

```diff
@@ -1,47 +1,74 @@
 (async () => {
-    function clear_all_games_table(){
+    function clear_table(){
         const table = document.getElementById('table');
         table.innerText = "";
     }
 
-async function create_all_games_table() {
-    clear_all_games_table();
-    const user = document.getElementById("user").value;
-    fetch('/all_games?user=' + user)
-        .then(async response => {
-            if (response.ok) {
-                const games = await response.json();
-                const table = document.getElementById('table')
-                games.forEach(addGamesToTable)
+    async function create_table(games){
```

Comment:
This whole function just seems like a lot.  Maybe it's clunky and could be slimmer and neater

Reply:
this is why we need react lol. This is very low level. As the application grows, it will become unmaintainable.